crates/biome_js_formatter/src/js/bindings/formal_parameter.rs

```diff
@@ -34,9 +37,16 @@ impl FormatNodeRule<JsFormalParameter> for FormatJsFormalParameter {
         let is_hug_parameter = node
             .syntax()
             .grand_parent()
-            .and_then(FormatAnyJsParameters::cast)
+            .and_then(JsParameters::cast)
             .map_or(false, |parameters| {
-                should_hug_function_parameters(&parameters, f.comments()).unwrap_or(false)
+                should_hug_function_parameters(
+                    &FormatAnyJsParameters::new(
+                        AnyJsParameters::JsParameters(parameters),
+                        FormatJsParametersOptions::default(),
+                    ),
+                    f.comments(),
+                )
+                .unwrap_or(false)
```

Comment:
there might be a more Rust-y way to do this, but i'm not at all sure. it can't just be `cast` any more since the `options` field of the struct now has to be specified.